products/cloudfunctions/inspec.yaml

```diff
@@ -12,6 +12,11 @@
 # limitations under the License.
 
 --- !ruby/object:Provider::Inspec::Config
+overrides: !ruby/object:Overrides::ResourceOverrides
+  CloudFunction: !ruby/object:Overrides::Inspec::ResourceOverride
+    properties:
+      trigger_http: !ruby/object:Overrides::Inspec::PropertyOverride
+        exclude: true
```

Comment:
Why do we want to exclude this in InSpec?

Reply:
It's not a real field. You tell CF to use a HTTP Trigger by sending an empty dictionary as the triggerHttps field.

That's super unintuitive, so Terraform added a `trigger_http` field on the handwritten version. I'll add a comment to make that more intuitive.